neuralmonkey/runners/regression_runner.py

```diff
@@ -15,22 +17,23 @@ class RegressionRunner(BaseRunner):
 
     def __init__(self,
                  output_series: str,
-                 decoder: Any,
-                 postprocess: Callable[[float], float]=None) -> None:
+                 decoder: SequenceRegressor,
+                 postprocess: Callable[[float], float] = None) -> None:
         super(RegressionRunner, self).__init__(output_series, decoder)
+        assert check_argument_types()
 
         self._postprocess = postprocess
 
     def get_executable(self,
-                       compute_losses: bool=False,
+                       compute_losses: bool = False,
                        summaries=True) -> Executable:
 
         if compute_losses:
             fetches = {"mse": self._decoder.cost}
         else:
             fetches = {}
 
-        fetches["prediction"] = self._decoder.decoded_logit
+        fetches["prediction"] = self._decoder.predicted
```

Comment:
Mně to přijde dobrý takhle. Za něčím, co by se jmenovalo `decoded` bych čekal něco disktrétního, ne float.